SU2_CFD/src/output_structure.cpp

```diff
@@ -8883,6 +8892,920 @@ void COutput::SetEquivalentArea(CSolver *solver_container, CGeometry *geometry,
   
 }
 
+void COutput::WriteSurface_Analysis(CConfig *config, CGeometry *geometry, CSolver *FlowSolver) {
```

Comment:
A question related to that: is it still necessary to define a double sided boundary for this marker ? We thought of maybe using this implementation for the FWH solver until we have the general extraction (which apparently is more complicated than I thought originally, @economon we have to talk about this soon).